src/components/Registration.js

```diff
@@ -2,13 +2,22 @@ import React from "react";
 import { Button } from "@material-ui/core";
 
 const Registration = () => (
-  <Button
-    href="http://seium21.eventbrite.pt/"
-    color="primary"
-    className="registration"
-  >
-    Inscrições
-  </Button>
+  <div className="registration-container">
+    <Button
+      href={"https://codingcompetitions.withgoogle.com/hashcode/"}
+      color="primary"
+      className="registration"
+    >
+      Hash Code
+    </Button>
+    <Button
+      href={"http://seium21.eventbrite.pt/"}
+      color="primary"
+      className="registration"
```

Comment:
What do you think about moving this links to a `config.json` (or yaml) file into `data/` folder?